src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -2752,7 +2755,7 @@ static void M2<T>(T t, dynamic? d)
         t = d;
     }
 }";
-            var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
+            var comp = CreateCompilation(source, options: WithNonNullTypesTrue(), parseOptions: TestOptions.Regular8);
```

Comment:
what motivated changing parseOptions to Regular8 here instead of pasting in the new diagnostics? or perhaps verifying behavior on both Regular8 and default?